package.json

```diff
@@ -1,10 +1,14 @@
 {
   "name": "eslint-config-springworks",
-  "version": "2.6.0",
+  "version": "2.7.0",
   "description": "ESLint configurations used by Springworks",
   "main": "index.js",
   "author": "Springworks",
   "license": "MIT",
+  "scripts": {
+    "lint": "eslint ./example.js",
+    "test": "NODE_ENV=test eslint ./example.js"
```

Comment:
Maybe have one example file for each config and test like this `eslint -c mocha-config.json examples/test-example.js`